cypress/e2e/awx/administration/credentialTypes.cy.ts

```diff
@@ -0,0 +1,290 @@
+      });
+    });
+
+    it('edit a credential type from the list row action and delete it using the list kebab menu', () => {
+      cy.navigateTo('awx', 'credential-types');
+      const editedCredentialTypeName = (credType1.name ?? '') + ' edited';
+      cy.filterTableByMultiSelect('name', [credType1.name]);
+      cy.clickTableRowPinnedAction(credType1.name, 'edit-credential-type', false);
+      cy.verifyPageTitle('Edit Credential Type');
+      cy.url().then((currentUrl) => {
+        expect(currentUrl.includes('edit')).to.be.true;
+      });
+      cy.get('[data-cy="name"]').clear().type(editedCredentialTypeName);
+      cy.get('[data-cy="description"]').clear().type('this is a new description after editing');
+      cy.intercept('PATCH', `api/v2/credential_types/${credType1.id}/`).as('editCredType');
+      cy.clickButton(/^Save credential type$/);
+      cy.wait('@editCredType')
+        .its('response.body.name')
+        .then((name: string) => {
+          expect(editedCredentialTypeName).to.be.equal(name);
+        });
+      cy.verifyPageTitle(editedCredentialTypeName);
+      cy.navigateTo('awx', 'credential-types');
+      cy.filterTableByMultiSelect('name', [editedCredentialTypeName]);
+      cy.clickTableRowKebabAction(`${editedCredentialTypeName}`, 'delete-credential-type', false);
+      cy.get('#confirm').click();
+      cy.intercept('DELETE', `api/v2/credential_types/${credType1.id}/`).as('deleteCredType');
+      cy.clickButton(/^Delete credential type/);
+      cy.contains(/^Success$/);
+      cy.clickButton(/^Close$/);
+      cy.wait('@deleteCredType').then((deleteCredType) => {
+        expect(deleteCredType?.response?.statusCode).to.eql(204);
+      });
+      cy.containsBy('.pf-v5-c-empty-state__title-text', 'No results found');
+      cy.verifyPageTitle('Credential Types');
+    });
+
+    it('can edit and delete a credential type from the details page', () => {
+      cy.navigateTo('awx', 'credential-types');
+      const editedCredentialTypeName = (credType1.name ?? '') + ' edited';
+      cy.filterTableByMultiSelect('name', [credType1.name]);
+      cy.clickTableRowLink('name', credType1.name, { disableFilter: true });
+      cy.clickButton('Edit credential type');
+      cy.verifyPageTitle('Edit Credential Type');
+      cy.get('[data-cy="name"]').clear().type(editedCredentialTypeName);
+      cy.get('[data-cy="description"]').clear().type('this is a new description after editing');
+      cy.intercept('PATCH', `api/v2/credential_types/${credType1.id}/`).as('editCredType');
+      cy.clickButton(/^Save credential type$/);
+      cy.wait('@editCredType')
+        .its('response.body.name')
+        .then((name: string) => {
+          expect(editedCredentialTypeName).to.be.equal(name);
+        });
+      cy.verifyPageTitle(editedCredentialTypeName);
+      cy.intercept('DELETE', `api/v2/credential_types/${credType1.id}/`).as('deleteCredType');
+      cy.clickPageAction('delete-credential-type');
```

Comment:
I could not find the assertion related to the edited items. Do we verify that after was saved the details page show the new name and new description?

Reply:
There was an assertion a couple of lines up; I added one additional.